### PR TITLE
more documentation

### PR DESCRIPTION
**Pull request by adrianmroz-allegro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,18 +72,18 @@ Install Turnilo distribution using [npm](https://www.npmjs.com/).
 npm install -g turnilo
 ```
 
-Start off by running an example with Covid-19 and Wikipedia datasets
+Start off by running turnilo with example datasets
```
Comment (mkuthan) [inline README.md:75]:
```suggestion
Start off by running Turnilo with example datasets
```

File: src/server/cli/options.ts [hunk 2]
```diff
@@ -15,11 +15,32 @@
...
+  `Port number to start server on. Default: ${DEFAULT_PORT}`
+).argParser(parseInteger);
+
+export const serverRootOption = new Option(
+  "--server-root <path>",
+  "Custom path to act as turnilo root"
+);
+export const serverHostOption = new Option(
+  "--server-host <hostname>",
+  "Hast that server will bind to"
```
Comment (mkuthan) [inline src/server/cli/options.ts:32]:
```suggestion
  "Host that server will bind to"
```

File: src/server/cli.ts [hunk 3]
```diff
@@ -116,8 +119,9 @@ program
 
 program
   .command("load-file")
-  .argument("<file>", "json file")
-  .requiredOption("-t, --time-attribute <attribute>", "time attribute")
+  .description("Runs turnilo and loads json file as a dataset")
```
Comment (mkuthan) [inline src/server/cli.ts:122]:
```suggestion
  .description("Runs Turnilo and loads json file as a dataset")
```

**Review by mkuthan — COMMENTED** [review]
What about https://allegro.github.io/turnilo/configuration-cluster.html?
See also: https://github.com/allegro/turnilo/search?q=%22--druid%22

File: README.md [hunk 4]
```diff
@@ -72,18 +72,18 @@ Install Turnilo distribution using [npm](https://www.npmjs.com/).
...
+Start off by running Turnilo with example datasets
 and open [http://localhost:9090/](http://localhost:9090/).
 
 ```
-turnilo --examples
+turnilo run-examples
 ```
 
-Or connect to the existing Druid broker using `--druid` command line option.
+Or connect to the existing Druid broker using `connect-druid` command.
```
Comment (piter75) [inline README.md:82]: Not the scope of this change but the sentence seems to be laid out better ;-)

```suggestion
Use `connect-druid` command to connect to the existing Druid broker.
```

**Review by piter75 — APPROVED** [review]

**Pull request merged**